Pillow security checks, remove pickle support, support xls in pandas, refactor API design to url_from/url_to (#1)
plotly/dash-io: README.md

        return (self.num == other.num) and (self.st == other.st)

obj = ExampleClass()
encoded = dio.url_from_pickle(obj)
decoded = dio.url_to_pickle(encoded)
```

## Documentation

You can access the documentation by calling:
```python
import dash_io as dio
help(dio)
```

> You can find the update to date output from `help` inside [`DOCS.txt`](DOCS.txt).

Reviewer: ```suggestion
You can find the up-to-date output from `help` inside [`DOCS.txt`](DOCS.txt).
```

        return (self.num == other.num) and (self.st == other.st)

obj = ExampleClass()
encoded = dio.url_from_pickle(obj)
decoded = dio.url_to_pickle(encoded)
```

#### Note

> If you are using Python 3.6, you will need to downgrade `pandas` to `0.24`, since pandas v0.25, v1.0 and v1.1 do not support in-memory pickle loading/saving. Once `pandas==0.24.*`, you will need to set compression to `None`:

Reviewer: ```suggestion
To use these pickle operations in Python 3.6, you will need to downgrade `pandas` to `0.24`, since pandas v0.25, v1.0 and v1.1 do not support in-memory pickle loading/saving. Once `pandas==0.24.*`, you will need to set compression to `None`:
```

Just to make it absolutely clear that the rest of this package can use the latest pandas.

Also: is `compression=None` required in both directions or just one? And these examples are supposed to be `url_to/from_pandas(format='pickle')`, right?

Now I'm thinking though perhaps we should remove `format='pickle'` as well - I believe `pd.read_pickle` has the same security implications as other unpickling operations, at root it's just calling [`pickle.load`](https://github.com/pandas-dev/pandas/blob/12513c4cdb14fe70ec7226e12bdea70faccdc2cc/pandas/io/pickle.py#L217) and I'm pretty sure that's all you need.

That would simplify the story around the pandas requirement too ;)

Author: That's a pretty good point. It's unfortunate tho, since pickle seems to be the most efficient pandas-only way to store a dataframe (parquet/feather require `pyarrow`, which is not included with the library, and CSV is very [slow](https://towardsdatascience.com/the-best-format-to-save-pandas-data-414dca023e0d)). 

If we remove it, maybe we should require `pyarrow` as a dependency and push people towards `feather` or `parquet`?

```python
encoded = dio.url_from_pickle(obj, compression=None)
decoded = dio.url_to_pickle(encoded, compression=None)
```

## Documentation

You can access the documentation by calling:
```python
import dash_io as dio
help(dio)
```

> Here's a (potentially outdated) output of the call above:

Reviewer: Not excited about writing "potentially outdated" in the readme 🙈 If you really want to put the help output here, [test it](https://www.pythonkitchen.com/get-pythons-help-function-stored-as-string-just-like-console/) instead.

```py
from pydoc import render_doc
def test_readme():
    with open(path_to_readme, 'r') as f:
        assert render_doc(dio) in f.read()
```
